my_list.rb

```diff
@@ -0,0 +1,11 @@
+class MyList
+  include MyEnumerable
```

Comment:
- [ ] I received errors when trying to run your code. You need to import the enumerable file here

  ```
  require_relative './my_enumerable.rb'
  ```
  ![my_list](https://user-images.githubusercontent.com/54848363/191936629-19ee1fd1-6efb-4046-b081-5c0e131d618e.jpg)